File: modal/modal.js
var mysql      = require('mysql');
var connection = mysql.createConnection({
  host     : 'localhost',
  user     : 'root',
  password : '',
  database : 'byjus_elevate'
});

connection.connect();

function GetTestSuit(){
    return new Promise((resolve, reject)=>{
        let q = 'SELECT * FROM batch_grades'
        connection.query(q, (error, result, fields)=>{
            if(error) reject(error);
            resolve(result);
        })
    })
}

module.exports = {GetTestSuit}

File: w_poc_2/modal.py
import mysql.connector

dataBase = mysql.connector.connect(
    host="localhost",
    user="root",
    passwd="",
    database="w_poc"
)

cursorObject = dataBase.cursor()

# query = "SELECT NAME FROM TEST_SUITE"
# cursorObject.execute(query)

# myresult = cursorObject.fetchall()

# for x in myresult:
#     print(x)


def setData(query, values):
    try:
        print('query>>', query, values)
        cursorObject.executemany(query, values)
        dataBase.commit()
        return True
    except Exception as err:
        print('error in setData', err)
        return False


def getData(query):
    cursorObject.execute(query)
    myresult = cursorObject.fetchall()
    return myresult
